### ISSUE TITLE
Position not updating realtime

### ISSUE THREAD

**Issue by Chrisnetika** [LeonBlade/BDTHPlugin#30]
Using version 1.4.1
I notice when I place somethin down and use the gizmo in rotate mode, the item does not update in realtime tho it does move once I place/confirm it.
Mostly happens with stuff that already placed I noticed.
Usually get thru it by moving the item with gizmo little by little otherwise i'm blinding moving it

**Comment by LeonBlade** [comment]
If you're outside this is going to happen, inside it happens on wall mounted items and certain other items.

**Comment by Chrisnetika** [comment]
> If you're outside this is going to happen, inside it happens on wall mounted items and certain other items.

anything I put on wall? like lofts and stuff right?

**Comment by LeonBlade** [comment]
Yes lofts count. I tried doing something that will show a preview I'll have to try it again sometime the last solution was flawed and crashed for some people.

**Comment by lodunost** [comment]
So even the wall is going to do this when I nudge them? I hope that a fix can be found. I truly love this tool, it is the best I have ever found.
